### PR TITLE
RS/JDJ/Rule19-18 bug fix

### PR DESCRIPTION
**Pull request by JacksonJ-KC**
The only part of the issue that I did not address is the RCTFailureException when ZERO.POWER is passed to table_G3_9_1_lookup if there are no supply fans. `RCTFailureException: The 'shaft_input_power' must be greater than 0.0 hp.` 

I'm not sure if this needs to be addressed, but wanted to bring attention to it.

### CODE REVIEW

File: rct229/rulesets/ashrae9012019/section19/section19rule18.py [hunk 1]
```diff
@@ -183,15 +183,16 @@ def get_calc_vals(self, context, data=None):
...
+                if exactly_one_supply_fan
                 else ZERO.POWER
             ).to(ureg.hp)
 
             A = 0.0
             more_than_one_exhaust_fan_and_energy_rec_is_relevant_b = False
             if (
                 fan_sys_b.get("air_energy_recovery")
-                and fan_sys_info_b["exhaust_fans_qty"] == 1
+                and fan_sys_b["air_energy_recovery"].get("type") != ENERGY_RECOVERY.NONE
```
Comment (weilixu) [inline rct229/rulesets/ashrae9012019/section19/section19rule18.py:194]: None does not equal to `ENERGY_RECOVERY.NONE`. Is that allowed here?

File: rct229/rulesets/ashrae9012019/section19/section19rule18.py [hunk 2]
```diff
@@ -172,9 +174,7 @@ def get_calc_vals(self, context, data=None):
             )
             supply_flow_b = fan_sys_info_b["supply_fans_airflow"].to(ureg.cfm)
 
-            more_than_one_supply_fan_b = (
-                fan_sys_info_b["supply_fans_qty"] != 1
-            )  # the reason why > 1 isn't used is to prevent an error when no supply fan exists
+            exactly_one_supply_fan = fan_sys_info_b["supply_fans_qty"] == 1
```
Comment (weilixu) [inline rct229/rulesets/ashrae9012019/section19/section19rule18.py:177]: Does no supply fan a concern in this rule? I saw it is captured by manual check so we should be fine with this logic.

**Review by weilixu — APPROVED** [review]
Thanks, I have two comments but generally this is a bug fix.

File: rct229/rulesets/ashrae9012019/section19/section19rule18.py (hunk 2 above)
Comment (JacksonJ-KC) [reply rct229/rulesets/ashrae9012019/section19/section19rule18.py:177]: This is related to my initial comment on the PR. I noticed in our test case for this scenario that the logic goes to `table_G3_9_1_lookup` and results in  `RCTFailureException: The 'shaft_input_power' must be greater than 0.0 hp.`

Comment (JacksonJ-KC) [reply rct229/rulesets/ashrae9012019/section19/section19rule18.py:177]: I am not sure if we want to handle this scenario more explicitly

Comment (weilixu) [reply rct229/rulesets/ashrae9012019/section19/section19rule18.py:177]: I see, the failure comes from missing supply flows - this is a typical data errors we should capture in the RCT. I do think it makes sense to leave it as is so the data look up function captures the issue.

**Pull request merged**